ftw/noticeboard/browser/configure.zcml

```diff
@@ -20,5 +20,13 @@
         class=".conditions.TermsAndConditions"
         />
 
+    <browser:page
+        for="ftw.noticeboard.content.noticeboard.INoticeBoard"
+        name="noticeboard_view"
```

Comment:
Do we have a preferred way of either using `-` or `_`? I noticed the previous routes in that pkg use `-`.

Reply:
I usually prefer `-`, but it's kinda a convention in plone to use `_view` for content type default views.